indexer/workers/importer.py

```diff
@@ -165,7 +166,11 @@ def process_story(self, sender: StorySender, story: BaseStory) -> None:
 
             # if publication date is none, fallback to rss_fetcher pub_date
             if data["publication_date"] is None:
-                data["publication_date"] = story.rss_entry()["pub_date"]
+                pub_date = story.rss_entry().pub_date
+                if pub_date is not None and pub_date != "":
```

Comment:
```suggestion
                if pub_date:
```
Should capture both `None` and `""`